packages/components/ng-ovh-payment-method/package.json

```
@@ -44,7 +44,7 @@
     "@ovh-ux/component-rollup-config": "^7.0.0"
   },
   "peerDependencies": {
-    "@ovh-ux/manager-config": "^1.0.0 || ^2.0.0",
+    "@ovh-ux/manager-config": "^0.4.0 || ^1.0.0",
```

```suggestion
    "@ovh-ux/manager-config": "^1.0.0 || ^2.0.0",
```
Maybe the revert should not revert package version...
Current `@ovh-ux/manager-config` version is `1.1.1` (https://github.com/ovh/manager/blob/develop/packages/manager/modules/config/package.json#L3) but will be updated to `2.0.0` in the next release with https://github.com/ovh/manager/commit/014d810e21f79b0ebf50d0aaf133328d37c4ba4b.

---

As discussed, these reverts can be omitted due to the fact that a fix is also required regarding this Pull Request.